databricks/koala/__init__.py

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+__version__ = "0.0.5"
```

Comment:
Is there a way to pull this value into `setup.py`? Otherwile they will get out of sync

Reply:
Possible but it's a bit hacky:

https://github.com/apache/spark/blob/5cdb8a23df6f269d6be0bf3536e9af9e29c4a05f/python/setup.py#L31-L37

Let me try.